src/valkey-cli.c

```diff
@@ -2273,6 +2273,8 @@ static void cliWaitForMessagesOrStdin(void) {
             /* Ctrl-C pressed */
             config.blocking_state_aborted = 0;
             config.pubsub_mode = 0;
+            printf("Exiting current connection, ready to reconnect to Valkey server... \n");
```

Comment:
```suggestion
            printf("Closing current connection. Ready to reconnect to Valkey server... \n");
```

Reply:
Receive a ctrl C, re-establish the connection?

i am not an expert on it, both are ok.

Reply:
I think the user knows he pressed Ctrl-C. What he doesn't know is that valkey-cli closes the connection and opens a new one.

Re-establish, reconnect, I don't know. Same same.